src/pretalx/orga/views/settings.py

```diff
@@ -181,7 +181,119 @@ def dispatch(self, request, event, pk):
         return redirect(request.event.orga_urls.team_settings)
 
 
-class InvitationView(ActionFromUrl, FormView):
+class EventReview(ActionFromUrl, FormView):
+    form_class = ReviewSettingsForm
+    template_name = 'orga/settings/review.html'
+
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs['obj'] = self.request.event
+        kwargs['attribute_name'] = 'settings'
+        kwargs['locales'] = self.request.event.locales
+        return kwargs
+
+    def form_valid(self, form):
+        form.save()
+        return super().form_valid(form)
+
+    def get_success_url(self) -> str:
+        messages.success(self.request, _('Your settings have been saved.'))
+        return reverse('orga:settings.review.view', kwargs={'event': self.request.event.slug})
+
+    def get_context_data(self, *args, **kwargs):
+        ctx = super().get_context_data(*args, **kwargs)
+        ctx['team'] = User.objects.filter(
+            permissions__is_reviewer=True,
+            permissions__event=self.request.event,
+        )
+        ctx['pending'] = EventPermission.objects.filter(event=self.request.event, user__isnull=True, is_reviewer=True)
+        return ctx
+
+
+class EventReviewInvite(View):
+
+    def post(self, request, event):
+        nick = request.POST.get('nick')
```

Comment:
This method is kind of long and has an early return in the middle, which makes it more confusing. Maybe it can be split into two parts: `_invite_existing_user()` and `_invite_new_user()`?

Reply:
Done.